models/intel/index.md

```diff
@@ -265,6 +265,16 @@ Deep Learning compressed models
 | [machine-translation-nar-en-ru-0001](./machine-translation-nar-en-ru-0001/description/machine-translation-nar-en-ru-0001.md) | 23.17 | 69.29 |
 | [machine-translation-nar-ru-en-0001](./machine-translation-nar-ru-en-0001/description/machine-translation-nar-ru-en-0001.md) | 23.17 | 69.29 |
 
+## Text To Speech
+
+Deep Learning models for speech synthesis (mel spectrogram generation and wave form generation).
+
+| Model Name                                                                                | Complexity (GFLOPs)  | Size (Mp)  |
+|-------------------------------------------------------------------------------------------|--------------------- |----------- |
+| [text-to-speech-en-0001](./text-to-speech-en-0001/description/text-to-speech-en-0001.md)  |                      |            |
+| forward-tacotron-duration-att-0001                                                        | 15.84                | 13.569     |
+| forward-tacotron-regression-att-0001                                                      | 7.65                 | 4.96       |
+| melgan-upsample-0001                                                                      | 48.38                | 12.77      |
```

Comment:
The names of the components of a composite model should start with the name of that composite model.

Reply:
Like this:
text-to-speech-en-0001-forward-tacotron-duration-att
text-to-speech-en-0001-forward-tacotron-regression-att
text-to-speech-en-0001-melgan-upsample

?

Reply:
That's better, but it seems unnecessarily verbose. Component names are generally supposed to describe the function of that particular component. For the first two it seems most logical to use the same names as in `forward-tacotron`:

* `text-to-speech-en-0001-duration-prediction`
* `text-to-speech-en-0001-regression`

I don't know what the best way to describe the last component is (`generation`, maybe?), but it should be something similar.

Reply:
Done